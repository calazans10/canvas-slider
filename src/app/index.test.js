import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '.';

describe('App', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
