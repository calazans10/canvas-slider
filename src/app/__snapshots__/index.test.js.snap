// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<MainWrapper>
  <MainHeader />
  <SliderSection
    height={400}
    images={
      Array [
        "0.jpg",
        "1.jpg",
        "2.jpg",
        "3.jpg",
        "4.jpg",
        "5.jpg",
        "6.jpg",
      ]
    }
    width={640}
  />
</MainWrapper>
`;
